curriculum/challenges/english/20-upcoming-python/learn-regular-expressions-by-building-a-password-generator/656ef54070c72161e6feb90f.md

```diff
@@ -0,0 +1,56 @@
+---
+id: 656ef54070c72161e6feb90f
+title: Step 22
+challengeType: 20
+dashedName: step-22
+---
+
+# --description--
+
+It seems all fine, but it would be nice to have a way to check that the generated password complies to specific features. For example, a minimum number of special characters, digits, or uppercase/lowercase letters. You are going to take care of that soon.
```

Comment:
```suggestion
It seems all fine, but it would be nice to have a way to check that the generated password complies to specific features. For example, a minimum number of special characters, digits, or uppercase/lowercase letters. You are going to take care of that very soon.
```